pkgs/development/interpreters/python/hooks/pytest-check-hook.sh

```diff
@@ -37,7 +39,9 @@ function pytestCheckPhase() {
         disabledTestsString=$(_pytestComputeDisabledTestsString "${disabledTests[@]}")
       args+=" -k \""$disabledTestsString"\""
     fi
-    for file in ${disabledTestFiles[@]}; do
+
+    eval "__internalDisabledTestFilesArray=($disabledTestFiles)"
```

Comment:
This does not seem like the right way to undo `lib.escapeShellArgs`...

Reply:
I think this is about right. At least I do not know a better method.

Only it should be done where `declare -ar disabledTestFiles`. That way, I would expect the variable to be converted immediately and transparently for the rest of the script and hooks. (Though I am not sure what happens if the hook gets called multiple times, if that can happen.)

And is it really necessary to use a different variable name. If yes, you would go the other way around, setting `__internalDisabledTestFilesArray` in the `buildPythonPackage` and use `disabledTestFiles` in the scripts as an array.